pkg/test/resource/v3/resource.go

```diff
@@ -288,6 +288,35 @@ func MakeRuntime(runtimeName string) *runtime.Runtime {
 	}
 }
 
+// MakeExtensionConfig creates a extension config for a cluster.
+func MakeExtensionConfig(mode string, extensionConfigName string, route string) *core.TypedExtensionConfig {
+	rdsSource := configSource(mode)
+
+	// HTTP filter configuration
+	manager := &hcm.HttpConnectionManager{
+		CodecType:  hcm.HttpConnectionManager_AUTO,
+		StatPrefix: "http",
+		RouteSpecifier: &hcm.HttpConnectionManager_Rds{
+			Rds: &hcm.Rds{
+				ConfigSource:    rdsSource,
+				RouteConfigName: route,
+			},
+		},
+		HttpFilters: []*hcm.HttpFilter{{
+			Name: wellknown.Router,
+		}},
+	}
+	pbst, err := ptypes.MarshalAny(manager)
```

Comment:
Can we move away from ptypes? It's been deprecated. I believe the replace here is:

```
anypb.New()
```

Reply:
I see `ptypes` is used everywhere in this file. Shall we change all the `ptypes` to `anypb` at some point in the future?

Reply:
Sure we can do that